daisy_workflows/export/disk_resizing_mon.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+METADATA_URL="http://metadata/computeMetadata/v1/instance"
+
+MAX_SIZE=$1
+BUFFER_SIZE=25
+INTERVAL=10
+
+echo "Max disk size ${MAX_SIZE}GB, buffer size ${BUFFER_SIZE}GB, starting monitoring available disk buffer every ${INTERVAL}s..."
+while [ 1 ]
```

Comment:
`while true; do`

Reply:
or better (since primary purpose of loop is action+sleep), `while sleep ${INTERVAL}; do`